### DISCUSSION TITLE
curl equivalents?

### DISCUSSION THREAD

**Discussion by elockman** [jasonacox/tinytuya#272] [Q&A] [answered]
Hello,

This library has been wonderful with quickly getting local control to my Tuya thermostat (PCT513-WB-TY).

This library is in python, but I was wondering if anyone understands it enough to provide curl format calls to the local ip address.

The Tuya developer site allows you to query devices with something like this:
```
curl --request GET "[https://openapi.tuyaus.com/v1.0/iot-03/devices/{deviceID}/status](https://openapi.tuyaus.com/v1.0/iot-03/devices/%7BdeviceID%7D/status)" --header "sign_method: HMAC-SHA256" --header "client_id: {clientID}" --header "t: 1674835464167" --header "mode: cors" --header "Content-Type: application/json" --header "sign: {signature}" --header "access_token: {token}"
```
Is there an equivalent call to the device locally?

**Comment by uzlonewolf** [comment, marked as answer]
Unfortunately the devices do not have a webserver built in and so there's no way to use curl with them.  tinytua does include a webserver that you can then call with curl, however it is very bare-bones and does not support fancy things like ThermostatDevice.
